Workaround for
24737747 Need to allow mixing older POSIX/SUS versions with C99/C11 compilers

Can't send upstream as is without breaking builds on Solaris 11.3 & older
systems that don't support XPG7 - need to find better solution for there.

diff --git a/configure.ac b/configure.ac
index d0e48d8c..5b9f5f2e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1434,7 +1434,7 @@ case $host_os in
         # ... this opt-in to get sockaddr_in6 and sockaddr_storage...
         CFLAGS="$CFLAGS -D__EXTENSIONS__"
         # ... and this opt-in to get file descriptor passing support
-        CFLAGS="$CFLAGS -D_XOPEN_SOURCE=500"
+        CFLAGS="$CFLAGS -D_XOPEN_SOURCE=700"
         ;;
 esac
 
